packages/component-header/src/components/HeaderMain/NavbarContainer/DropdownItem/index.js

```diff
@@ -54,7 +53,7 @@ const LinkItem = ({ link, dropdownName, handleLinkEvent }) => (
       href={link.href}
       onClick={e => handleLinkEvent(e, link)}
       onKeyDown={e => handleLinkEvent(e, link)}
-      onFocus={() => trackGAEvent({ text: link.text, component: dropdownName })}
+      onClick={() => trackGAEvent({ text: link.text, component: dropdownName })}
```

Comment:
onClick on lines 54 and 56 - triggers no duplicate props linting error. Need just one. Assuming it's 56.